crates/symbolicator-service/src/services/sourcemap_lookup.rs

```diff
@@ -342,7 +342,10 @@ pub fn join_paths(base: &str, right: &str) -> String {
     }
 
     for seg in segments {
-        write!(final_path, "/{seg}").unwrap();
+        // FIXME: do we want to skip `.` even if we have a non-http scheme?
```

Comment:
For file paths this should be fine, right? Is there another scheme where it might be a problem?

Reply:
A few lines above, we have:

```
if right_segment == "." && (is_http || is_first_segment) {
    continue;
}
```

So we are skipping only leading `./` when joining paths. I wonder why we had this restriction? But if we want to be faithful, we gotta skip the `.` only in the http case…